app/models/user.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  attr_accessor :remember_token
+  validates :name, presence: true, uniqueness: { case_sensitive: false }
+  has_many :creator_events, foreign_key: 'creator_id', class_name: 'Event'
+
+  has_many :attendances, foreign_key: 'attendee_id'
+  has_many :attended_events, through: :attendances, class_name: 'Event'
+
+  scope :upcoming, ->(id) { Event.joins(:attendances).where('attendee_id == ?', id).upcoming }
+  scope :past, ->(id) { Event.joins(:attendances).where('attendee_id == ?', id).past }
```

Comment:
Please, remove these because you already created the new one in the Event model.